docs/reference/elasticsearch/configuration-reference/elasticsearch-settings.md

```diff
@@ -0,0 +1,148 @@
+---
+navigation_title: "{{es}} settings for ECH and ECE"
+mapped_pages:
+  - https://www.elastic.co/guide/en/cloud/current/ec-add-user-settings.html#ec-es-elasticsearch-settings
+  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-add-user-settings.html#ece-change-user-settings-examples
+applies_to:
+  deployment:
+    ess: 
+    ece:
+---
+
+# Configure {{es}} [add-user-settings]
```

Comment:
> # Configure {{es}} [add-user-settings]

I think this content (previewed in https://docs-v3-preview.elastic.dev/elastic/elasticsearch/pull/125000/reference/elasticsearch/configuration-reference/elasticsearch-settings) looks the same as what's in https://docs-v3-preview.elastic.dev/elastic/docs-content/tree/main/deploy-manage/deploy/elastic-cloud/edit-stack-settings and https://docs-v3-preview.elastic.dev/elastic/docs-content/tree/main/deploy-manage/deploy/cloud-enterprise/edit-stack-settings-elasticsearch IMO that deployment section is the right place for these how-tos, so we should remove the redundant duplication from here.